src/redux/operationsAuth.ts

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+import { BaseRoutes } from "../interfaces-submodule/enums/routes/base-routes.enum";
+import { AuthRoutes } from "../interfaces-submodule/enums/routes/auth-routes.enum";
+import { ILoginRequestDTO } from "../interfaces-submodule/interfaces/dto/auth/iadmin-login-request.interface";
+import { ILoginResponseDTO } from "../interfaces-submodule/interfaces/dto/auth/ilogin-response.interfaces";
+import { RootState } from "./store";
+
+axios.defaults.baseURL = 'https://trainee-api.chat.abcloudz.com';
```

Comment:
- [ ] if you use axios, create it as a service with methods you need and use it for the app